### PR TITLE
fixed most of transcript of practical ai 27

### PR DESCRIPTION
**Pull request by zeddee**
fixed most unintelligible segments (except 2); fixed one typo 'treat'->'treatment'

### CODE REVIEW

File: practicalai/practical-ai-27.md [hunk 1]
```diff
@@ -72,9 +72,9 @@ In the prospective studies not only is the individual first informed what it is
...
 
-In the first case, when we create features, we have results showing that we can discriminate parkinsonian patients that are on the medication \[unintelligible 00:16:34.23\] or off the medication, using features that include frequency components of the voice that are not detectable by the human ear, but they are still there because the drug is psychoactive, so it affects your network system, and of course, trivially, it affects your voice.
+In the first case, when we create features, we have results showing that we can discriminate \[between\] parkinsonian patients that are on the medication (Levodopa) or off the medication, using features that include frequency components of the voice that are not detectable by the human ear, but they are still there because the drug is psychoactive, so it affects your network system, and of course, trivially, it affects your voice.
```
Comment (pnevares) [inline practicalai/practical-ai-27.md]: I haven't seen an example of corrected context like the `\[between\]` you've added, but I can see an argument for it in this case where discrimination means something else without it. I'll tag @jerodsanto to make sure it's kosher.

Comment (jerodsanto) [reply practicalai/practical-ai-27.md]: I dig it 👍

**Comment by jerodsanto** [comment]
@all-contributors  please add @zeddee for doc

**Comment by zeddee** [comment]
thanks for accepting y'all 🎉  have a good year ahead!

**Pull request merged**